performance-test/src/puppeteer-test.ts

```diff
@@ -0,0 +1,362 @@
+        xref: 'x',
+        yref: 'y',
+        x0: 16.6,
+        x1: 16.6,
+        y0: 0,
+        y1: 100,
+        line: {
+          color: 'rgb(245, 66, 66)',
+          width: 1,
+          dash: 'dash',
+        },
+      },
+      {
+        type: 'line',
+        xref: 'x',
+        yref: 'y',
+        x0: 33.33,
+        x1: 33.33,
+        y0: 0,
+        y1: 100,
+        line: {
+          color: 'rgb(168, 50, 149)',
+          width: 1,
+          dash: 'dash',
+        },
+      },
+      {
+        type: 'line',
+        xref: 'x',
+        yref: 'y',
+        x0: 66.6,
+        x1: 66.6,
+        y0: 0,
+        y1: 100,
+        line: {
+          color: 'rgb(26, 255, 0)',
+          width: 1,
+          dash: 'dash',
+        },
+      },
+      {
+        type: 'line',
+        xref: 'x',
+        yref: 'y',
+        x0: 133.3,
+        x1: 133.3,
+        y0: 0,
+        y1: 100,
+        line: {
+          color: 'rgb(255, 239, 0)',
+          width: 1,
+          dash: 'dash',
+        },
+      },
+      {
+        //Quartile25 Line below
+        type: 'line',
+        xref: 'x',
+        yref: 'y',
+        x0: frameData.percentile25,
+        x1: frameData.percentile25,
+        y0: 0,
+        y1: 100,
+        line: {
+          color: 'rgb(0, 0, 0)',
+          width: 1,
+          dash: 'solid',
+        },
+      },
+      {
+        //Quartile50 Line below
+        type: 'line',
+        xref: 'x',
+        yref: 'y',
+        x0: frameData.percentile50,
+        x1: frameData.percentile50,
+        y0: 0,
+        y1: 100,
+        line: {
+          color: 'rgb(0, 0, 0)',
+          width: 1,
+          dash: 'solid',
+        },
+      },
+      {
+        //Quartile75 Lines below
+        type: 'line',
+        xref: 'x',
+        yref: 'y',
+        x0: frameData.percentile75,
+        x1: frameData.percentile75,
+        y0: 0,
+        y1: 100,
+        line: {
+          color: 'rgb(0, 0, 0)',
+          width: 1,
+          dash: 'solid',
+        },
+      },
+    ],
+    yaxis: {
+      showgrid: true,
+      zeroline: true,
+      showline: true,
+      range: [0, 35],
+      autotick: false,
+      ticks: 'outside',
+      tick0: 0,
+      dtick: 5,
+      ticklen: 8,
+      tickwidth: 2,
+      tickcolor: '#000',
+      title: {
+        text: 'Frequency',
+        font: {
+          family: 'Courier New, monospace',
+          size: 10,
+          color: '#7f7f7f',
+        },
+      },
+    },
+  }
+  const imgOpts = {
+    format: 'png',
+    width: 700,
+    height: 500,
+  }
+  const figure = { data: [trace], layout: layout }
+
+  return new Promise<string>((resolve, reject) => {
+    plotly.getImage(figure, imgOpts, async function (error: any, imageStream: any) {
+      if (error) return console.log(error)
+
+      var fileStream = fs.createWriteStream(testFileName)
+      imageStream.pipe(fileStream)
+      const path1 = path.resolve(testFileName)
+      const path2 = path.resolve('src')
```

Comment:
I think the images should be in a different directory that is not src. and that directory should be put to gitignore. because I _think_ running the script now would result in untracked files in git, right?